<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="testcases.CreateAccountPOM" tests="1" time="30.367" ignored="3" failures="0" timestamp="21 Apr 2018 15:07:36 GMT" skipped="0" hostname="INDIA-VAISHNAVK" errors="1">
  <testcase name="createAccount" time="30.367" classname="testcases.CreateAccountPOM">
    <error type="java.lang.RuntimeException" message="Unexpected result for screenshot command: null">
      <![CDATA[java.lang.RuntimeException: Unexpected result for screenshot command: null
	at org.openqa.selenium.remote.RemoteWebDriver.getScreenshotAs(RemoteWebDriver.java:300)
	at wdMethods.SeMethods.takeSnap(SeMethods.java:362)
	at utils.Reporter.reportStep(Reporter.java:49)
	at utils.Reporter.reportStep(Reporter.java:70)
	at wdMethods.SeMethods.selectDropDownUsingText(SeMethods.java:187)
	at pages.AccountsPage.selectIndustry(AccountsPage.java:46)
	at testcases.CreateAccountPOM.createAccount(CreateAccountPOM.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createAccount -->
  <testcase name="afterTest" classname="wdMethods.ProjectMethods">
    <skipped/>
  </testcase> <!-- afterTest -->
  <testcase name="beforeSuite" classname="wdMethods.ProjectMethods">
    <skipped/>
  </testcase> <!-- beforeSuite -->
  <testcase name="afterSuite" classname="wdMethods.ProjectMethods">
    <skipped/>
  </testcase> <!-- afterSuite -->
</testsuite> <!-- testcases.CreateAccountPOM -->
